src/main/java/mil/dds/anet/resources/ReportResource.java

```diff
@@ -744,4 +769,51 @@ public Response showRollupEmail(@Auth Person user, @QueryParam("startDate") Long
 			return Utils.resultGrouper(list, "stats", "personId", tlf);
 		}
 	}
+	
+	/**
+	 * The comparator to be used when ordering the roll up graph results to ensure
+	 * that any pinned organisation names are returned at the start of the list.
+	 * 
+	 * @author timothy.ward
+	 *
+	 */
+	public static class RollupGraphComparator implements Comparator<RollupGraph> {
+
+		private final List<String> pinnedOrgNames;
+
+		/**
+		 * Creates an instance of this comparator using the supplied pinned organisation
+		 * names.
+		 * 
+		 * @param pinnedOrgNames
+		 *            the pinned organisation names
+		 */
+		public RollupGraphComparator(final List<String> pinnedOrgNames) {
+			this.pinnedOrgNames = pinnedOrgNames;
+		}
+
+		/*
+		 * (non-Javadoc)
+		 * 
+		 * @see java.util.Comparator#compare(java.lang.Object, java.lang.Object)
+		 */
+		@Override
+		public int compare(final RollupGraph o1, final RollupGraph o2) {
+
+			if (pinnedOrgNames.contains(o1.getOrg().getLongName())) {
+				if (pinnedOrgNames.contains(o2.getOrg().getLongName())) {
+					return pinnedOrgNames.indexOf(o1.getOrg().getLongName())
+							- pinnedOrgNames.indexOf(o2.getOrg().getLongName());
+				} else {
+					return -1;
+				}
+			} else if (pinnedOrgNames.contains(o2.getOrg().getLongName())) {
+				return 1;
+			} else {
+				return 0;
```

Comment:
shouldn't this, instead of 0, be something like o1.getOrg().getLongName().compareTo(o2.getOrg().getLongName()) ?

Reply:
And if we want it to be the same as the sorting in `client/src/pages/rollup/Show.js` it should also use the shortName here.